common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java

```diff
@@ -56,10 +60,26 @@ protected void onConfigUpdate(Config<?> config) {
     }
 
     private Pattern buildPattern() {
-        return Pattern.compile(
-                "(?<!\\[)\\b(" + McUtils.mc().getUser().getName()
-                        + (!aliases.get().isEmpty() ? "|" + aliases.get().replace(",", "|") : "") + ")\\b(?!:|])",
-                Pattern.CASE_INSENSITIVE);
+        if (aliases.get().isEmpty()) {
+            return Pattern.compile(
+                    "(?<!\\[)\\b" + McUtils.mc().getUser().getName() + "\\b(?!:|])", Pattern.CASE_INSENSITIVE);
+        } else {
+            // we can filter the provided aliases and remove the invalid parts, then tell the user
+            String[] splitAliases = aliases.get().split(",");
+            List<String> filteredAliases = Arrays.stream(splitAliases)
+                    .filter(alias -> !alias.isEmpty())
+                    .map(alias -> FILTERED_CHARACTERS.matcher(alias).replaceAll(""))
+                    .toList();
+
+            if (splitAliases.length != filteredAliases.size()) {
```

Comment:
I'm not sure I understand this. You check that there are as many items in both lists? How does that indicate incorrect characters?

Reply:
This only indicates a parse fail, so something like `,,`. Other characters are filtered out automatically and silently. I think bolyai's comment was also regarding this

Reply:
Hm. For this to work, I think the filter and map calls need to be in the opposite order, right?

Have you tested that this work?

Reply:
Even if the order was reversed, that wouldn't change the list length, it would just be an empty string where applicable

It's not intended/designed to catch when some invalid char is replaced, just when an entire alias parse fails

Reply:
If you do a 
```
   .filter(alias -> !alias.isEmpty())
```
after replacing all invalid characters with the empty string, then I think it would, no? That means that an input string like "OK1,,$@,OK2" would result in the list "OK1", "OK2", which is different from the list "OK1", "", "$@", "OK2".

Otherwise I do not understand what you mean by "an entire alias parse fail".